bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -227,6 +234,7 @@ public String getHtmlScreenReaderText() {
      */
     @Override
     @Nullable
+    @JsonSerialize(using = EncodeHTMLSerializer.class)
```

Comment:
this would affect model.json output also.

Reply:
ack